daemon/graphdriver/imagerestorer.go

```diff
@@ -0,0 +1,30 @@
+package graphdriver
+
+import (
+	"github.com/docker/docker/image"
+	"github.com/docker/docker/pkg/archive"
+)
+
+// NOTE: These interfaces are used for implementing specific features of the Windows
+// graphdriver implementation.  The current versions are a short-term solution and
+// likely to change or possibly be eliminated, so avoid using them outside of the Windows
+// graphdriver code.
```

Comment:
Unfortunately, we can't, since there is common code that needs to use these interfaces in daemon.go (see line 687).